js/main.js

```diff
@@ -23,4 +31,100 @@ function handleSubmit(event) {
   data.entries.unshift(newObject);
   $img.setAttribute('src', 'images/placeholder-image-square.jpg');
   $form.reset();
+  $ul.prepend(renderTree(newObject));
+  handleClick();
+}
+
+function renderTree(newObject) {
+  var $li = document.createElement('li');
+
+  var $rowOne = document.createElement('div');
+  $rowOne.setAttribute('class', 'row');
+  $li.appendChild($rowOne);
+
+  var $columnHalfOne = document.createElement('div');
+  $columnHalfOne.setAttribute('class', 'column-half');
+  $rowOne.appendChild($columnHalfOne);
+
+  var $img = document.createElement('img');
+  $img.setAttribute('src', newObject.url);
+  $columnHalfOne.appendChild($img);
+
+  var $columnHalfTwo = document.createElement('div');
+  $columnHalfTwo.setAttribute('class', 'column-half');
+  $rowOne.appendChild($columnHalfTwo);
+
+  var $rowTwo = document.createElement('div');
+  $rowTwo.setAttribute('class', 'row');
+  $columnHalfTwo.appendChild($rowTwo);
+
+  var $h3 = document.createElement('h3');
+  $h3.setAttribute('class', 'margin-top-0');
+  $h3.textContent = newObject.title;
+  $rowTwo.appendChild($h3);
+
+  var $rowThree = document.createElement('div');
+  $rowThree.setAttribute('class', 'row');
+  $columnHalfTwo.appendChild($rowThree);
+
+  var $p = document.createElement('p');
+  $p.textContent = newObject.notes;
+  $rowThree.appendChild($p);
+
+  return $li;
+}
+
+function handleLoad(event) {
+  for (var i = 0; i < data.entries.length; i++) {
+    var value = renderTree(data.entries[i]);
+    $ul.appendChild(value);
+  }
+}
+
+if (data.view === 'entries') {
+  handleClick();
+} else if (data.view === 'entry-form') {
+  handleNew();
+}
+
+function handleClick(event) {
+  data.view = 'entries';
+  if (data.view === 'entries') {
+    switchView();
+  }
+}
+
+function handleNew(event) {
+  data.view = 'entry-form';
+  if (data.view === 'entry-form') {
+    switchView();
+  }
+}
+
+function switchView(event) {
+  if (data.entries.length === 0) {
+    $noEntries.className = 'row none';
+  } else {
+    $noEntries.className = 'row hidden none';
+  }
+  if (data.view === 'entry-form') {
+    for (var i = 0; i < $input.length; i++) {
+      var view = $input[i].getAttribute('data-view');
+      if (data.view === view) {
+        $input[i].className = 'input';
+      } else {
+        $input[i].className = 'input hidden';
+      }
+    }
+  }
+  if (data.view === 'entries') {
+    for (var j = 0; j < $input.length; j++) {
+      var setView = $input[j].getAttribute('data-view');
+      if (data.view === setView) {
+        $input[j].className = 'input';
+      } else {
+        $input[j].className = 'input hidden';
+      }
+    }
+  }
```

Comment:
Both of these conditionals have the same work in them, you should be able to just do the work, no conditionals necessary 🤔